### ISSUE TITLE
Is it dead?

### ISSUE THREAD

**Issue by ameeno** [petrstepanov/gnome-macos-remap-wayland#1]
Is this project dead?

Wanna know before i install, the other project that linked me to this one is X11 Xorg only, and obviously i'm not gonna replace my display manager to an antique just to have some mac bindings on my fedora. (More likely to make mac linux like/windows like, as my keyboard is pc anyway.)

However, i noticed there has not been any updates on this project in 5 months, dont wanna install something thats no longer active :)

**Comment by petrstepanov** [comment]
Hi @ameeno this repository is not dead, it is in early development. I've attempted to try [Key Mapper](https://github.com/sezanzeb/key-mapper) and [xremap](https://github.com/k0kubun/xremap) options for remapping but did not succeed with a stable result yet. I'll let you know when solid working option will be available.

**Comment by jackywu** [comment]
great job @petrstepanov , I'm looking forward of that.

**Comment by jacksongoode** [comment]
@petrstepanov Still really interested in the project!

**Comment by ccidral** [comment]
@petrstepanov Just came by to show my appreciation for your project. I used a hackintosh for years on an old Lenovo laptop, but since Apple is locking out Intel users, I decided to give Ubuntu a try. And because I'm 100% spoiled by the Apple magic keyboard and the MacOS shortcuts, [gnome-macos-remap](https://github.com/petrstepanov/gnome-macos-remap) saved my life. I've been using it for almost a year now. Thank you for it! ❤️ I hope the wayland version succeeds.
